app/src/main/java/edu/wisc/ece/pinpoint/pages/map/MapFragment.java

```diff
@@ -221,4 +242,32 @@ private void getDeviceLocation() {
             });
         }
     }
+
+    private void setPinnieCount() {
+        FirebaseDriver driver = FirebaseDriver.getInstance();
```

Comment:
We already have a FirebaseDriver declared in this class as `firebase`, so maybe just use that here instead of getting the instance again.